snapcraft.yaml

```diff
@@ -74,12 +74,14 @@ parts:
       craftctl default
     override-prime: |
       # Do nothing
+
   splash-theme:
     plugin: dump
     source: https://github.com/snapcore/plymouth-theme-ubuntu-core.git
     source-type: git
     organize:
       ubuntu-core: usr/share/plymouth/themes/ubuntu-core
+
```

Comment:
Please don't add unnecessary white-space changes. It increases the chance of merge conflicts when we pull in changes from the core-base repo.

Reply:
fixed.